### ISSUE TITLE
Changing month and year

### ISSUE THREAD

**Issue by SergeiStruk** [naxeem/Pikaday-Material#2]
User doesn't have ability to change quickly month and year. It will be hard to use it for date of birth field.

**Comment by naxeem** [comment]
Yes, I removed the ability to to do that for a specific use case... You should not use this plugin when you need to pick dob, etc... I may update the plugin with the ability to do that in the future..
